tests/operator/operator.go

```diff
@@ -127,14 +127,10 @@ var _ = Describe("[Serial][sig-operator]Operator", Serial, decorators.SigOperato
 	var vmYamls map[string]*vmYamlDefinition
 
 	var (
-		patchKvProductNameVersionAndComponent  func(string, string, string, string)
```

Comment:
```
drop several patch lambdas
* patchKvProductNameVersionAndComponent
* patchKvVersionAndRegistry
* patchKvVersion
* patchKvCertConfig
```

Would you add a word about *why* this is good, and how it's done? it's not the bare-simple movement of func.